api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/RdfLiteralHash.java

```diff
@@ -63,7 +67,13 @@ public static int makeRdfLiteralHash( DataPropertyStatement stmt ){
             log.debug("got hash " + hashMe.hashCode() + " for String '" + hashMe + "'");
         return hashMe.hashCode();
     }
-
+    
+    private static String replaceIntegerWithInt(String predicate) {
+        if( predicate.equals("http://www.w3.org/2001/XMLSchema#integer")) {
```

Comment:
Can you define constants for http://www.w3.org/2001/XMLSchema#integer and http://www.w3.org/2001/XMLSchema#int? It might be also useful somewhere else. Also, I am not sure we need method here, maybe ternary operator can be used instead (langOrDatatype.equals(XML_INTEGER_TYPE))?XML_INT_TYPE:langOrDatatype.

Reply:
I didn't create constants as I think that this code is deprecated and should be removed after we replace n3editing with dynamic API. It is a separate method for the same reasons: to emphasize that it is a temporary fix and make it easy to remove it in future. Also ternary operators harder to read and less practical for debugging.